main/import/hearThis/test/scenarios/index.ts

```diff
@@ -1,5 +0,0 @@
-import { scenario1 } from './scenario1';
-import { scenario2 } from './scenario2';
-import { scenario3 } from './scenario3';
-
-export const scenarios = [scenario1, scenario2, scenario3];
```

Comment:
Are all these _import_ scenarios now covered by _source_ test scenarios?

Reply:
These three scenarios are testing going from a `convertProject` to a `BKProject` which no longer happens. And they use the same test data that `fullImport.integration` uses (`main/test`) so just having the full import test should cover them.